src/model/zod.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+export enum TaskStatus {
+  IN_PROGRESS = 'IN_PROGRESS',
+  TODO = 'TODO',
+  COMPLETE = 'COMPLETE',
+}
+
+export const taskSchema = z.object({
```

Comment:
CapitalCase is often used for schema vars in zod because it aligns with naming conventions for classes and types in TypeScript, and schema is practically just a blueprint for validating an object type.

```suggestion
export const TaskSchema = z.object({
```